.github/workflows/test.yml

```
@@ -34,3 +34,8 @@ jobs:
 
     - name: Test
       run: make test-coverage
+
+    - name: Verify Examples
```

The existing (above) we could change to something like `Test Coverage`.

```suggestion
    - name: Test
```